src/Client/WebSocketClient.php

```diff
@@ -15,10 +15,10 @@
 use Nekland\Woketo\Message\MessageHandlerInterface;
 use Nekland\Woketo\Rfc6455\FrameFactory;
 use Nekland\Woketo\Rfc6455\MessageFactory;
-use Nekland\Woketo\Rfc6455\MessageHandler\CloseFrameHandler;
-use Nekland\Woketo\Rfc6455\MessageHandler\PingFrameHandler;
-use Nekland\Woketo\Rfc6455\MessageHandler\RsvCheckFrameHandler;
-use Nekland\Woketo\Rfc6455\MessageHandler\WrongOpcodeHandler;
+use Nekland\Woketo\Rfc6455\FrameHandler\CloseFrameHandler;
+use Nekland\Woketo\Rfc6455\FrameHandler\PingFrameHandler;
+use Nekland\Woketo\Rfc6455\FrameHandler\RsvCheckFrameHandler;
+use Nekland\Woketo\Rfc6455\FrameHandler\WrongOpcodeHandler;

```

Comment:
Why WrongOpcode is not a FrameHandler in its name ?

And why naming CloseFrameHandler that while we have the information in the directory name